Utils/ScriptValidator/Unit1.pas

```diff
@@ -30,21 +34,37 @@   TForm1 = class(TForm)
 
 var
   Form1: TForm1;
-
-
+  IsValidatePath : Boolean;
+  sListFileInFolder : TStringList;
 implementation
 uses
   KM_Maps;
 
 {$R *.dfm}
 
+procedure TForm1.FindFiles(Path: String; out aList: TStringList);
+var SearchRec:TSearchRec;

```

Comment:
Please mind the code formatting. This should be:

var
  SearchRec: TSearchRec;